mmomni/ansible/playbooks/jitsi/turnserver.conf

```diff
@@ -0,0 +1,31 @@
+stream {
+    upstream web {
+        server 127.0.0.1:4444;
+    }
+    upstream turn {
+        server 127.0.0.1:5349;
+    }
+    # since 1.13.10
+    map $ssl_preread_alpn_protocols $upstream {
```

Comment:
We found this works very poorly in practice. I'd recommend using a dedicated subdomain for TURN and running it on port 443, using SNI to differentiate the traffic.

Reply:
Yes, the problem is adding another extra domain complicates a bit the thing. I going to leave it as is now, and rethink about it later.

Reply:
You can then run it on a different port, it will work on most scenarios, except some very strict corporate networks.

Reply:
Yes, that is our main concern. I think this is better to keep this for now, and we can change it later adding another domain, at least optionally.